src/Build/BackEnd/Components/SdkResolution/SdkResolverLoader.cs

```diff
@@ -57,6 +66,13 @@ internal class SdkResolverLoader
             return resolvers.OrderBy(t => t.Priority).ToList();
         }
 
+        internal virtual IList<SdkResolverManifest> GetResolversManifests(LoggingContext loggingContext,
+            ElementLocation location)
+        {
+            return FindPotentialSdkResolversManifests(
+                Path.Combine(BuildEnvironmentHelper.Instance.MSBuildToolsDirectory32, "SdkResolvers"), location);
```

Comment:
```suggestion
                Path.Combine(BuildEnvironmentHelper.Instance.MSBuildToolsDirectoryRoot, "SdkResolvers"), location);
```

(this is the new one we invented to avoid some confusion, today it has the same value as `ToolsDirectory32` but it communicates intent more clearly)